task-list/config/routes.rb

```diff
@@ -1,7 +1,6 @@
 Rails.application.routes.draw do
   get '/' => 'tasks#index'
-  get 'tasks/:id' => 'tasks#show'
-
+  resources :tasks
```

Comment:
We are _not_ using resources _yet_ so please update your routes to use the manual RESTful routes